README.md

```diff
@@ -0,0 +1,13 @@
+
+# Optimism batch decoder
+Decodes the calldata of a batcher transaction and returns the L2 transactions within it
+
+## Usage
+- install the repo
+- use `testWithExampleData` function to decode example calldata. You can edit `example-data/calldata.txt` to test different calldata
+- use `decodeBatcherTransaction` to only provide the transaction hash and provider URL. It will fetch the calldata for you and decode the same way.
+
+NB! SpanBatch decoding is unimplemented at the moment.
+
+
+Made with 💛 at Blocktorch
```

Comment:
Let's add the prerequisites to run the service + ready made command to run it

Reply:
Made it into a npm package and added install command